java/build.gradle

```diff
@@ -18,6 +18,7 @@ allprojects {
         google()
         jcenter()
         maven { url 'https://esri.jfrog.io/artifactory/arcgis' }
+        maven { url 'https://olympus.esri.com/artifactory/arcgisruntime-repo/' }
```

Comment:
Let's keep this line out of main. Since it's our internal artifactory, it's not of use to our users

Reply:
Good catch! I cannot make changes to this repo since it is protected. I will close this PR and create a new one with this line removed!